includes/forum.php

```diff
@@ -139,6 +139,7 @@ class AsgarosForum {
         'view_name_markallread'            => 'markallread',
         'view_name_members'                => 'members',
         'view_name_profile'                => 'profile',
+		'view_name_editprofile'			   => 'edit',
```

Comment:
@glorious1 I think the view name here should be `editprofile` instead of `edit`?

Reply:
Yes of course.  No need to ask my opinion; I was just copying your code like a monkey.  I didn't even understand much of it.